doghouse/server/cookieman/cookieman.go

```diff
@@ -0,0 +1,109 @@
+package cookieman
+
+import (
+	"encoding/base64"
+	"net/http"
+)
+
+type Cipher interface {
+	Encrypt(plaintext []byte) ([]byte, error)
+	Decrypt(ciphertext []byte) ([]byte, error)
+}
+
+type CookieMan struct {
+	defaultOpt CookieOption
+	cipher     Cipher
+}
+
+type CookieOption struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type CookieOption should have comment or be unexported